Docs/reference/content/reference/driver/crud/client_side_encryption.md

```diff
@@ -181,4 +181,153 @@ namespace MongoDB.Driver.Examples
 }
 ```
 
-**Coming soon:** An example using the community version and demonstrating explicit encryption/decryption.
+### Explicit Encryption and Decryption
+
+Explicit encryption and decryption is a **MongoDB community** feature and does not use the `mongocryptd` process. Explicit encryption is provided by the `ClientEncryption` class. The following example has been adapted from [`ExplicitEncryptionExamples.cs`](https://github.com/mongodb/mongo-csharp-driver/blob/master/tests/MongoDB.Driver.Examples/ExplicitEncryptionExamples.cs):
```

Comment:
Should `community` be capitalized and should "Edition" be added?

    **MongoDB Community Edition**

Do we really need a link to the `ExplicitEncryptionExamples.cs` class? Isn't this documentation complete and does not require that link?

Reply:
I suggested the change because the official product name is "MongoDB Community Server":

https://www.mongodb.com/download-center/community

Reply:
Same for "MongoDB Enterprise Server":

https://www.mongodb.com/download-center/enterprise

Reply:
Done